### ISSUE TITLE
Recursive bulk-add/bulk-search

### ISSUE THREAD

**Issue by Tricky12321** [dularion/streama#691]
### Issue description
My movies are stored in a single folder, with a sub folder for each movie like this:
/mnt/.../movies/MovieName.1080p.bla-bla/MovieName.1080p.bla-bla.mp4
When i add the moves folder in buld-add /mnt/.../movies/, it searches only the top folder, and not recursive into the folders.

### Steps to Reproduce

1. Bulk-add a folder, with sub folders containing movie
2. Nothing happens.

### Expected Behaviour
I was expecting it to search all sub folders recursively 

### Actual Behaviour

Searches the top folder, and ignores content of sub folders

### Environment Information

- **Operating System**: Arch Linux
- **Streama version**: v1.6.0-RC9

**Comment by HannahRy** [comment]
Currently this is the behaviour. Recursiveness is planned.

**Comment by Blueblazer172** [comment]
Also waiting for this. I have a folder structure as following:
```
SeriesName
     --[SeasonNumber]_SeasonName
           --[EpisodeNumber]_EpisodeName
```

**Comment by varac** [comment]
Yes please ! How else would I initially import hundreds of files organised like the examples here in this issue ? This is a show stopper :/

**Comment by dellarb** [comment]
Another vote for this. And massive thanks for everyone involved :)

**Comment by cappannonno** [comment]
Anyone found a way to bypass this issue? I have all my library stored with Plex subfolder method and I cannot use Streama :(

**Comment by MagicD3VIL** [comment]
Is this on the Roadmap? For a high priority issue, it has been over a year unsolved

**Comment by majidf** [comment]
Is there a technical issue that hinders implementation of this feature?

**Comment by cmullins83** [comment]
This. It is seriously a PITA having to add each episode of each season to a show when you have hundreds of shows.
Movies too, but that's not quite as terrible as there's usually just one file per movie.
